src/sandbox.rkt

```diff
@@ -38,6 +40,56 @@
       ([(pt ex) (in-pcontext context)])
     (values pt ex)))
 
+(define (get-exacts test pts)
```

Comment:
All of this code is _very_ ugly but I also don't see it as a merge blocker. It's new code, it doesn't make anything worse, and getting everything merged will help us with further work.

Reply:
Sandbox is going to need a major clean up. I think, ideally, sandbox provides multiple run modes that mirror the web API that perform the same setup, e.g., setting the seed, and cleanup

Reply:
Yep, I think @zaneenders and @bryncurry are potentially interested in this as a bigger project.